src/main/kotlin/com/team4099/robot2023/subsystems/superstructure/Superstructure.kt

```diff
@@ -0,0 +1,990 @@
+        }
+      }
+      SuperstructureStates.SINGLE_SUBSTATION_INTAKE_CONE -> {
+        manipulator.currentRequest =
+          Request.ManipulatorRequest.TargetingPosition(
+            Manipulator.TunableManipulatorStates.singleSubstationIntakeShelfExtension.get(),
+            Manipulator.TunableManipulatorStates.coneInVoltage.get()
+          )
+        // Outputs
+
+        // Transition
+        if (manipulator.isAtTargetedPosition) {
+          nextState = SuperstructureStates.IDLE
+        }
+      }
+      SuperstructureStates.SCORE_PREP -> {
+        groundIntake.currentRequest =
+          Request.GroundIntakeRequest.TargetingPosition(
+            GroundIntake.TunableGroundIntakeStates.stowedDownAngle.get(),
+            GroundIntake.TunableGroundIntakeStates.neutralVoltage.get()
+          )
+        if (groundIntake.isAtTargetedPosition) {
+          val scoreHeight =
+            when (nodeTier) {
+              NodeTier.HYBRID -> {
+                when (usingGamePiece) {
+                  GamePiece.CUBE ->
+                    Elevator.TunableElevatorHeights.hybridHeight.get() +
+                      Elevator.TunableElevatorHeights.cubeDropPosition.get()
+                  GamePiece.CONE ->
+                    Elevator.TunableElevatorHeights.hybridHeight.get() +
+                      Elevator.TunableElevatorHeights.coneDropPosition.get()
+                  else -> 0.0.inches
+                }
+              }
+              NodeTier.MID -> {
+                when (usingGamePiece) {
+                  GamePiece.CUBE ->
+                    Elevator.TunableElevatorHeights.midCubeHeight.get() +
+                      Elevator.TunableElevatorHeights.cubeDropPosition.get()
+                  GamePiece.CONE ->
+                    Elevator.TunableElevatorHeights.midConeHeight.get() +
+                      Elevator.TunableElevatorHeights.coneDropPosition.get()
+                  else -> 0.0.inches
+                }
+              }
+              NodeTier.HIGH -> {
+                when (usingGamePiece) {
+                  GamePiece.CUBE ->
+                    Elevator.TunableElevatorHeights.highCubeHeight.get() +
+                      Elevator.TunableElevatorHeights.cubeDropPosition.get()
+                  GamePiece.CONE ->
+                    Elevator.TunableElevatorHeights.highConeHeight.get() +
+                      Elevator.TunableElevatorHeights.coneDropPosition.get()
+                  else -> 0.0.inches
+                }
+              }
+              else -> 0.0.inches
+            }
+          val extension =
+            when (nodeTier) {
+              NodeTier.HYBRID -> Manipulator.TunableManipulatorStates.lowScoreExtension.get()
+              NodeTier.MID -> Manipulator.TunableManipulatorStates.midScoreExtension.get()
+              NodeTier.HIGH -> Manipulator.TunableManipulatorStates.highScoreExtension.get()
+              else -> 0.0.inches
+            }
+          manipulator.currentRequest =
```

Comment:
you actually do need to sequence these -- if the manipulator extends before the elevator (or even at the same time, potentially) it will clip the lower nodes as it goes up